galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/main/java/dev/galasa/zos3270/internal/Zos3270ManagerImpl.java

```diff
@@ -207,4 +210,8 @@ protected IDynamicStatusStoreService getDss() {
     public IZosManagerSpi getZosManager() {
         return this.zosManager;
     }
+    
+    public ITextScannerManagerSpi getTextScannerManager() {
```

Comment:
I'm reviewing this on iPhone so might have missed the answer. Why do we need a getter method in the manager if we are passing the instance to the implementation object?